src/builder/arg_group.rs

```diff
@@ -188,6 +188,21 @@ impl ArgGroup {
         self
     }
 
+    /// Getters for all args. It will return a vector of `Id`
+    ///
+    /// Example
+    ///
+    /// ```rust
+    /// # use clap::{ArgGroup};
+    /// let group = ArgGroup::new("myprog")
+    ///     .args(["f", "c"]);
+    ///
+    /// assert_eq!(group.get_args(), ["f", "c"]);
+    /// ```
+    pub fn get_args(self) -> Vec<Id> {
+        self.args
+    }
+
```

Comment:
Thanks a lot for reviews ^^

> These functions should be taking `&str` rather than destroying the ArgGroup.

Do you mean `&mut self`? otherwise I probably miss something here ^^ 

>  The return type should then either be impl Iterator<Item=Id> or impl Iterator<Item=&Id>

Make sense 🙌

Reply:
Oops, yes, `&self` (no `mut` as they are read-only)